src/components/form/__spec__.js

```diff
@@ -407,101 +411,100 @@ describe('Form', () => {
       let buttonContainers;
 
       beforeEach(() => {
-        buttons = TestUtils.scryRenderedDOMComponentsWithTag(instance, 'button')
-        buttonContainers = TestUtils.scryRenderedDOMComponentsWithTag(instance, 'div');
+        wrapper = shallow(<Form />)
+        buttons = wrapper.find(Button)
+        buttonContainers = TestUtils.scryRenderedDOMComponentsWithTag(wrapper, 'div');
       });
 
-      it('renders two buttons', () => {
-        expect(buttons.length).toEqual(2);
-      });
-
-      it('renders a secondary cancel button with cancelClasses', () => {
-        TestUtils.findRenderedDOMComponentWithClass(instance, 'carbon-form__cancel');
-        expect(buttons[1].className).toMatch('carbon-button carbon-button--secondary');
-      });
-
-      it('when cancelText prop is passed it renders the secondary button with the prop', () => {
-        instance = TestUtils.renderIntoDocument(
-          <Form cancelText={'Foo'} />
-        );
-        buttons = TestUtils.scryRenderedDOMComponentsWithTag(instance, 'button')
-        expect(buttons[1].innerHTML).toEqual('Foo');
-      });
-
-      it('when cancelText prop is not passed it renders the secondary button with default text', () => {
-        expect(buttons[1].innerHTML).toEqual(
-          I18n.t('actions.cancel', { defaultValue: 'Cancel' })
-        );
-      });
-
-      it('renders a primary save button with saveClasses', () => {
-        expect(buttons[0].className).toMatch('carbon-button carbon-button--primary');
-        expect(buttonContainers[1].className).toEqual('carbon-form__save');
-      });
-
-      it('renders an undisabled save button if not submitting', () => {
-        expect(buttons[0].disabled).toBeFalsy();
-      });
-
-      it('renders a disabled save button if saving', () => {
-        instance = TestUtils.renderIntoDocument(
-          <Form saving={true} />
-        );
-        buttons = TestUtils.scryRenderedDOMComponentsWithTag(instance, 'button')
-        expect(buttons[0].disabled).toBeTruthy();
-      });
-
-      describe('when saveButtonProps is passed', () => {
-        it('sets save button props', () => {
-          let theme = 'magenta';
-
-          instance = TestUtils.renderIntoDocument(<Form saveButtonProps={ { theme: theme } } />);
-          buttons = TestUtils.scryRenderedDOMComponentsWithTag(instance, 'button')
-          expect(buttons[0].className).toContain(`carbon-button--${theme}`);
+      describe('the save button', () => {
+        it('by default it renders a save button', () => {
+          expect(wrapper.find('.carbon-form-save')).toBeTruthy();
```

Comment:
this will always return true as find returns an array every time. You need to do something like 
`expect(wrapper.find('.carbon-form-save').exists()).toBeTruthy();`